load_game.py

```diff
@@ -549,17 +549,19 @@ def load_baaj():
     FFXC.set_neutral()
     memory.main.wait_frames(30 * 0.04)
 
+
 def kilika_rng_manip():
     # Kilika start, RNG01
     # 1904657448
     logger.warning("==== Hard setting value for testing")
     rng_value = 1904657448
     memory.main.set_rng_by_index(value=rng_value, index=1)
-    
-    #Basically, hunt until we don't find a good battle in 'advances'
+
+    # Basically, hunt until we don't find a good battle in 'advances'
     advances = 5
-    import rng_track
     import area.kilika
+    import rng_track
```

Comment:
It might make more sense to pull these up. But testing is a thing that isn't really done well here. So feel free to ignore this one.